Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -252,14 +299,26 @@ additional lines added to the :file:`composer.json` from above:
          }
       }
 
-   There is no harm in that, but it won't have any effect.
+   There is no harm in having that, but it won't have any effect.
+
+Ideally you should not manually edit a :file:`composer.json` file, but instead
+utilize the Composer command to manipulate the file, like this:
+
+.. code-block:: shell
+   :caption: typo3_root$
+
+   composer config repositories.foo-bar vcs https://github.com/foo/bar.git
```

Comment:
The repositories are not introduced above, so why mentioning it here? This may be confusing.

Reply:
It's an example how to programattically adapt the "repositories" instead of editing the file. Will check if I can move it or mention more specifically why tht example is given.